File: ToptanTrackRepository/src/main/java/com/ahbap/ToptanTrack/TotalPriceDto.java
package com.ahbap.ToptanTrack;

import lombok.AllArgsConstructor;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;
import lombok.ToString;

import java.math.BigDecimal;

@EqualsAndHashCode
@ToString
@AllArgsConstructor
public class TotalPriceDto {
    public BigDecimal totalPrice;
    public String name;
    public int stock;
}

File: ProductApp/app/src/main/java/com/ahbap/android/app/productapp/retrofit/dto/TotalPriceDto.kt
package com.ahbap.android.app.productapp.retrofit.dto

import kotlinx.serialization.Serializable
import java.math.BigDecimal


data class TotalPriceDto(var totalPrice : BigDecimal =  BigDecimal.ZERO,
                            var name : String = "",
                            var stock : Int = 0
    )
